### PR TITLE
Update Settings for FP 18

### PR DESCRIPTION
**Pull request by SlavaRa**

### CODE REVIEW

**Comment by elsassph** [comment]
Looks like we stopped including the corresponding `playerglobal.swc`s in `Tools/flexlibs` since FP14. Should we fix that?

File: FlashDevelop/Bin/Debug/Projects/310 Haxe - Flash Project/Project.hxproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <movie fps="30" />
     <movie width="800" />
     <movie height="600" />
-    <movie version="17" />
+    <movie version="18" />
```
Comment (elsassph) [inline FlashDevelop/Bin/Debug/Projects/310 Haxe - Flash Project/Project.hxproj]: We need to check what FP version is supported by the Haxe compiler - Haxe 3.1.3 didn't even accept FP17.

**Comment by Meychi** [comment]
We do patch the appman installed SDKs. The FD bundled SWCs are if we patch any used SDK. They add the size of FD. What should we do?

File: FlashDevelop/Bin/Debug/Projects/310 Haxe - Flash Project/Project.hxproj (hunk 1 above)
Comment (Meychi) [inline FlashDevelop/Bin/Debug/Projects/310 Haxe - Flash Project/Project.hxproj]: Last supported version for haxe 3.2 is 17 = 28. 3.1.3 supports also up to 12.5 -> 28 but swf-version is labeled incorrectly. Last correct version is 12.0 -> 23. What should we do?

**Comment by Meychi** [comment]
I will merge this and lets fix the Haxe versions before release.

**Comment by elsassph** [comment]
My opinion is that we should use the most commonly supported Flash version in the Haxe (and possibly Flash) templates. Someone who really needs a specific more recent Flash version will (probably) know what he is doing.

**Comment by Meychi** [comment]
Then that would be 12.

**Pull request merged**